test/exe/Main.hs

```diff
@@ -1854,6 +1854,18 @@ outlineTests = testGroup
                               (R 0 0 0 10)
                               [docSymbol "C" SkConstructor (R 0 9 0 10)]
       ]
+  , testSessionWait "record fields" $ do
+    let source = T.unlines ["data A = B {", "  x :: Int", "  , y :: Int}"]
+    docId   <- openDoc' "A.hs" "haskell" source
```

Comment:
This needs rebasing after your `lsp-test` bump has been merged. CI is correctly failing because this symbol no longer exists.